app/assets/stylesheets/foreman_virt_who_configure/config.css.scss

```diff
@@ -15,3 +15,20 @@ pre.terminal {
  .terminal-buttons {
```

Comment:
I know it's not you'r code, but since you're changing the file, could you please fix the indentation here too?